trace/discovery_gtrace.go

```diff
@@ -6,15 +6,15 @@ import (
 	"context"
 )
 
-// discoveryComposeOptions is a holder of options
+// discoveryComposeOptions is a holder of options.
```

Comment:
need to fix internal/cmd/gtrace tool because *_gtrace.go files are generated with tool